### PR TITLE
fix broken link

### PR DESCRIPTION
**Pull request by dungeonmaster18**
It fixes the broken links on the create image page.

### CODE REVIEW

**Comment by dungeonmaster18** [comment]
PTAL @mistyhacks

File: engine/userguide/eng-image/baseimages.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ Most Dockerfiles start from a parent image. If you need to completely control
 the contents of your image, you might need to create a base image instead.
 Here's the difference:
 
-- A [parent image](/reference/glossary.md#parent-image) is the image that your
+- A [parent image](/glossary.md?term=parent image) is the image that your
```
Comment (mistyhacks) [inline engine/userguide/eng-image/baseimages.md]: Almost there. Encode the space: `?term=parent%20image`

File: engine/userguide/eng-image/baseimages.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ Most Dockerfiles start from a parent image. If you need to completely control
...
   image is based on. It refers to the contents of the `FROM` directive in the
   Dockerfile. Each subsequent declaration in the Dockerfile modifies this parent
   image. Most Dockerfiles start from a parent image, rather than a base image.
   However, the terms are sometimes used interchangeably.
 
-- A [base image](/reference/glossary.md#base-image) either has no `FROM` line
+- A [base image](/glossary.md?term=base image) either has no `FROM` line
```
Comment (mistyhacks) [inline engine/userguide/eng-image/baseimages.md]: As above.

Comment (dungeonmaster18) [reply engine/userguide/eng-image/baseimages.md]: Ping @mistyhacks applied changes.

**Pull request merged**